File: specification/specs/indices/alias_management/get_alias/GetAliasRequest.ts
@rest_spec_name('indices.get_alias')
class GetAliasRequest extends RequestBase {
  path_parts?: {
    name?: Names
    index?: Indices
  }
  query_parameters?: {
    allow_no_indices?: boolean
    expand_wildcards?: ExpandWildcards
    ignore_unavailable?: boolean
    local?: boolean
  }
  body?: {}
}

File: output/csharp/Indices/AliasManagement/GetAlias/GetAliasRequest.cs
using Nest.Common;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Indices {

	public class GetAliasRequest : RequestBase {
		
		[DataMember(Name="allow_no_indices")]
		public bool AllowNoIndices { get; set; }


		[DataMember(Name="expand_wildcards")]
		public ExpandWildcards ExpandWildcards { get; set; }


		[DataMember(Name="ignore_unavailable")]
		public bool IgnoreUnavailable { get; set; }


		[DataMember(Name="local")]
		public bool Local { get; set; }

	}
}
